frappe/templates/print_formats/standard_macros.html

```diff
@@ -136,8 +136,10 @@
 		and frappe.utils.is_image(doc[df.fieldname]) %}
 		<img src="{{ doc[df.fieldname] }}" class="img-responsive"
 			{%- if df.print_width %} style="width: {{ get_width(df) }};"{% endif %}>
-    {% elif df.fieldtype=="HTML" %}
-        {{ frappe.render_template(df.options, {"doc":doc}) }}
+	{% elif df.fieldtype=="HTML" %}
+		{{ frappe.render_template(df.options, {"doc":doc}) }}
+	{% elif df.fieldtype=="Currency" %}
+		{{ doc.get_formatted(df.fieldname, doc, translated=df.translatable) }}
 	{% else %}
 		{{ doc.get_formatted(df.fieldname, parent_doc or doc, translated=df.translatable) }}
```

Comment:
For table fields, `doc` is child document
parent_doc is given preference over doc